src/Hit/ColorTerminal.hs

```diff
@@ -115,19 +116,24 @@ arrow = " ➤  "
 -- | Represents a user's answer
 data Answer = Y | N
 
--- | Parse an answer to 'Answer'
-yesOrNo :: Text -> Maybe Answer
-yesOrNo (T.toLower -> answer )
-    | T.null answer = Just Y
+-- | Parse an answer to 'Answer'. Takes a default `Answer` to return in case of empty user input.
+yesOrNoWithDefault :: Answer -> Text -> Maybe Answer
+yesOrNoWithDefault def (T.toLower -> answer )
+    | T.null answer = Just def
     | answer `elem` ["yes", "y", "ys"] = Just Y
     | answer `elem` ["no", "n"]  = Just N
     | otherwise = Nothing
 
-prompt :: IO Answer
-prompt = do
-    answer <- yesOrNo . T.strip <$> getLine
+yesOrNoText :: Answer -> Text
+yesOrNoText N = boldCode <> "y"   <> resetCode <> "/[n]"
```

Comment:
Let's make `[n]` bold instead :slightly_smiling_face:

Reply:
Oops, sorry about that. Missed it when I switched the order. All fixed. :)

Reply:
No worries!